### DISCUSSION TITLE
Trying to convert Chitubox to CXDLP file but was told that it was unable to detect the resolution.

### DISCUSSION THREAD

**Discussion by SamHenderson-1** [sn4k3/UVtools#682] [Q&A]
I just got my first 3d printer, a Creality Halot One. Since I heard that the Creality slicer is bad I decided to slice a exposure test model in Chitubox and then convert it in UVTools to a CXDLP. however, when I tried to convert it I got the following message: 
![image](https://user-images.githubusercontent.com/71032897/227663378-8eeb5c1e-d724-4d9a-ba0a-f5ac9a8aa550.png)
What could be going on here? Thanks in advance!

**Comment by sn4k3** [comment]
The resolution you defined on slicer is wrong. Must be 1620 x 2560 for your printer.
But you don't need to convert ctb to cxdlp, chitubox can export directly to cxdlp

**Reply by SamHenderson-1** [reply]
Ah! Thanks for the heads up! I don't see cxdlp in the save as type options
